creator-node/src/routes/files.js

```diff
@@ -105,6 +105,8 @@ module.exports = function (app) {
           cnodeUserUUID: req.session.cnodeUserUUID,
           multihash: fileResp.hash,
           sourceFile: fileResp.path,
+          dirMultihash: dirCID,
+          fileName: fileResp.path.split('/').slice(-1)[0] || fileResp.path,
```

Comment:
when is this `|| fileResp.path` case hit? first clause should always be truthy no?